### ISSUE TITLE
login failed

### ISSUE THREAD

**Issue by ursystempass** [borisuu/TelnetClient.php#1]
I use this package but when running, the error 'Login Failed' appears. 

I use php 8.1, composer version 2.4.1, laravel 10, and windows 11

**Comment by borisuu** [comment]
Can you try to connect via telnet via PowerShell? If that fails then you have a problem with your credentials, otherwise I'll check if the code is faulty in some way.
You can try the following:
```
telnet xxx.xxx.xxx.xxx
```
when prompted for the username and password, put them in.

**Comment by ursystempass** [comment]
> 

I've tried to connect via PowerShell and PuTTY, both went smoothly.

**Comment by borisuu** [comment]
Can you check the format of your password prompt? The default is 'Password:' and if that differs, you'll have to use a custom prompt.

**Comment by ursystempass** [comment]
> Can you check the format of your password prompt? The default is 'Password:' and if that differs, you'll have to use a custom prompt.

Yes, it remains the same as written in your package. Nothing has changed from the configuration you wrote, I followed everything in order

**Comment by borisuu** [comment]
Can you post the username and password prompts?

**Comment by ursystempass** [comment]
> Can you post the username and password prompts?

usn : admin
pass : rahasia 

hope this is exactly what you asked for

**Comment by borisuu** [comment]
So in your console you see:

```
usn :
pass :
```

**Comment by ursystempass** [comment]
> So in your console you see:
> 
> ```
> usn :
> pass :
> ```

I checked it and it is appropriate

**Comment by borisuu** [comment]
Add a dump statement right above this line and output the original error, this might help debugging:
https://github.com/borisuu/laravel-telnet/blob/928cefb8c0973afae54633c97986ab5b5d21447e/src/TelnetClient.php#L513
